utils/get-us-data.R

```diff
@@ -82,7 +82,7 @@ load_observed_deaths <- function(weekly = FALSE,
                   epiweek = as.numeric(paste0(epiweek, ".", as.numeric(day))))
   
   true_deaths_national <- true_deaths %>%
-    dplyr::group_by(date, epiweek) %>%
+    dplyr::group_by(date, epiweek, day, week) %>%
```

Comment:
seems odd to group by both epiweek and week.

Reply:
true, this was just so it remained a column after summarising

Reply:
if epiweek and week differ this will have odd results